Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -133,6 +133,12 @@ using SourceLinkAccessorDelegate =
     Delegate<std::pair<source_link_iterator_t, source_link_iterator_t>(
         const Surface&)>;
 
+/// expected max number of track states that are expected to be added by
+/// stateCandidateCreator
+/// @note if the number of states exceeds this number dynamic memory allocation will occur.
+//.       the number is chosen to yield a container size of 64 bytes.
+static constexpr std::size_t NMaxBranchesPerSurface = 10;
```

Comment:
```suggestion
static constexpr std::size_t s_maxBranchesPerSurface = 10;
```

I think this is what we usually use for globals in Acts